kairos-cli/build.rs

```diff
@@ -0,0 +1,20 @@
+use std::env;
+use std::fs;
+use std::path::Path;
+
+fn main() {
+    // Path
+    let session_binaries_dir = env::var("PATH_TO_SESSION_BINARIES")
+        .expect("PATH_TO_SESSION_BINARIES environment variable is not set");
+
+    // Get the output directory set by Cargo.
+    let out_dir = env::var("OUT_DIR").expect("OUT_DIR not set");
+    let source_path = Path::new(&session_binaries_dir).join("deposit-session-optimized.wasm");
+    let dest_path = Path::new(&out_dir).join("deposit-session-optimized.wasm");
+
+    // Copy the file from the source to the destination
+    fs::copy(&source_path, dest_path).expect("Failed to copy WASM file");
+
+    // Print out a message to re-run this script if the source file changes.
+    println!("cargo:rerun-if-changed={}", source_path.display());
```

Comment:
NIT: This works, but I am not clear why we need to copy the wasm into `OUT_DIR`. Why not just `include_bytes` from PATH_TO_SESSION_BINARIES directly?

Reply:
`include_bytes!(env!("PATH_TO_SESSION_BINARIES"))` would work too. However, I would like to extend build script to automatically discover and optimize WASMs (similar to [this](https://github.com/cspr-rad/kairos/blob/fdcd1f595d0067e37f936046097787c3c2cab666/demo-contract-tests/tests/test_fixture/wasm_helper.rs#L6)) so `PATH_TO_SESSION_BINARIES` can be avoided.

The whole project works great in Nix, but being able to just `cargo build` without bunch of extra env setup would be better.

Reply:
Sounds good